html-web-page/public/index.html

```diff
@@ -4,29 +4,41 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
 
-    <title>My Website</title>
+    <title>Romcal v3</title>
```

Comment:
this was updated to reflect the same state as https://romcal.js.org

Reply:
> this was updated to reflect the same state as https://romcal.js.org/

Where? I cannot see `Romcal v3` anywhere on the page, but I could find `Romcal` in here:

```html
<title>Liturgical calendars of the Catholic Roman Rite | Romcal</title>
```

---

I am for adding an `About`/`Info` page with the used romcal version (or in the footer, like it is now at romcal.js.org), however, should we really use it in the title? :thinking:

Reply:
apologies, i was thinking of https://github.com/romcal/romcal-examples/blob/main/html-web-page/index.html#L13, where it is already h1'd as `romcal v3`. so the casing differs on my side. Either way `My Website` doesn't seem appropriate.

Reply:
so, should we remove `v3` from both the h1 & title, or leave as-is, where they match (`romcal v3`)?

Reply:
i'm fine with this, but the title had been the same `romcal v3` long before i started touching it.

Reply:
True. I belive it was because Etinne started working on this when `v2` was considered the latest stable version, while `v3` was heavily under development. :man_shrugging: